.github/workflows/snyk-scan.yaml

```diff
@@ -0,0 +1,73 @@
+name: Snyk Security Vulnerability Scan
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+    branches:
+      - main
+
+jobs:
+  snyk_scan_test:
+    if: ${{ github.event_name == 'pull_request' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

Comment:
Why are we using `@master`?. This is not common. Use the major version tag or provide a explanation.

Reply:
I will use the major version.

Reply:
For snyk setup there is no major version (in docs also https://github.com/snyk/actions/tree/master/setup)